### PR TITLE
Eventpage draft

### PR DESCRIPTION
**Pull request by ramumaha**
Hey,the alignment with the card components and tab seems off.Please look at it.
Thanks.

### CODE REVIEW

**Comment by harisarang** [comment]
@ramumaha  Build seems to be failing. @sam9111 can you post the build logs here

File: src/components/Card.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React, { Component } from "react";
```
Comment (harisarang) [inline src/components/Card.js:1]: Remove unused `Component` here

File: src/components/Current.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+function Current() {
+  return (
+    <flex>
+      <div className="Current">
+        <Card
+          title=" Workshop on essentials for Coding Interview"
+          imageImport={vvcoding}
+          body="By Vrishin Vigneshwar SDE @Motorq"
+        />
+        <br></br>
```
Comment (harisarang) [inline src/components/Current.js:14]: Don't use `<br></br>` instead use `<br/>`

File: src/components/Current.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import Card from "./Card";
+import vvcoding from "../images/vvvcoding.svg";
+import carrer from "../images/careertalk.png";
```
Comment (harisarang) [inline src/components/Current.js:4]: Typo `career`

File: src/components/EventHeader.js [hunk 4]
```diff
@@ -0,0 +1,60 @@
+// src/pages/Events.js
+import React from "react";
+import { Heading, useColorModeValue } from "@chakra-ui/react";
+import { Tab, Tabs, TabList, TabPanel } from "react-tabs";
```
Comment (harisarang) [inline src/components/EventHeader.js:4]: Remove unused components

File: src/components/EventHeader.js [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+import "react-tabs/style/react-tabs.css";
+import { Box, Button, Flex, Spacer, Link } from "@chakra-ui/react";
```
Comment (harisarang) [inline src/components/EventHeader.js:6]: Remove unused components

File: src/pages/2020_2021.js [hunk 6]
```diff
@@ -0,0 +1,129 @@
+import React, { Component } from "react";
+import { chakra, Box, Flex, useColorModeValue, Link } from "@chakra-ui/react";
```
Comment (harisarang) [inline src/pages/2020_2021.js:2]: Remove unused components in import

File: src/pages/2020_2021.js [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+          borderRadius="lg"
+        >
+          <Box w={{ lg: "45%" }}>
+            <Box
+              h={{ base: 14, sm: "full" }}
+              rounded={{ lg: "lg" }}
+              bgSize="cover"
+              bgImage={vvcoding}
+              width={300}
+              bgSize="contain"
```
Comment (harisarang) [inline src/pages/2020_2021.js:34]: Duplicate prop is present here.
Are you using eslint ? As I have mentioned in [readme](https://github.com/sam9111/ssn-acm-w/blob/master/README.md)

File: src/pages/2020_2021.js [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
```
Comment (harisarang) [inline src/pages/2020_2021.js:88]: Duplicate prop is present here.
Are you using eslint ? As I have mentioned in [readme](https://github.com/sam9111/ssn-acm-w/blob/master/README.md)

File: src/pages/events.js [hunk 9]
```diff
@@ -1,9 +1,50 @@
 // src/pages/Events.js
-import React from "react";
-import { Box, Button } from "@chakra-ui/react";
-
-function EventsPage() {
-  return <Box>Events</Box>;
+import React, { Component } from "react";
+import { Heading, useColorModeValue } from "@chakra-ui/react";
```
Comment (harisarang) [inline src/pages/events.js:3]: Remove unused imports

File: src/pages/past.js [hunk 10]
```diff
@@ -0,0 +1,74 @@
...
+          borderRadius="lg"
+        >
+          <Box w={{ lg: "45%" }}>
+            <Box
+              h={{ base: 14, sm: "full" }}
+              rounded={{ lg: "lg" }}
+              bgSize="cover"
+              bgImage={vvcoding}
+              width={300}
+              bgSize="contain"
```
Comment (harisarang) [inline src/pages/past.js:33]: Duplicate prop is present here.
Are you using eslint ? As I have mentioned in [readme](https://github.com/sam9111/ssn-acm-w/blob/master/README.md)

File: src/pages/past.js [hunk 11]
```diff
@@ -0,0 +1,74 @@
+import React, { Component } from "react";
+import { chakra, Box, Flex, useColorModeValue, Link } from "@chakra-ui/react";
```
Comment (harisarang) [inline src/pages/past.js:2]: Remove unused components

File: src/pages/upcoming.js [hunk 12]
```diff
@@ -0,0 +1,18 @@
+import React, { Component } from "react";
+import { Box, Button, Flex } from "@chakra-ui/react";
```
Comment (harisarang) [inline src/pages/upcoming.js:2]: Remove unused components

**Review by harisarang — CHANGES_REQUESTED** [review]
Here are my thoughts, 

1. There are many unused imports
2. Having duplicate props in some props can really cause issue in production.
3. Make sure to setup eslint and prettier for the project. Check in [readme](https://github.com/sam9111/ssn-acm-w/blob/master/README.md)
4. There are `.png` and `.svg` for same file remove which are not necessary

File: src/components/Card.js (hunk 1 above)
Comment (ramumaha) [reply src/components/Card.js:1]: Removed

File: src/components/Current.js (hunk 2 above)
Comment (ramumaha) [reply src/components/Current.js:14]: Changed

File: src/components/Current.js (hunk 3 above)
Comment (ramumaha) [reply src/components/Current.js:4]: Resolved

File: src/components/EventHeader.js (hunk 4 above)
Comment (ramumaha) [reply src/components/EventHeader.js:4]: Removed.

File: src/components/EventHeader.js (hunk 5 above)
Comment (ramumaha) [reply src/components/EventHeader.js:6]: Removed.

File: src/pages/2020_2021.js (hunk 6 above)
Comment (ramumaha) [reply src/pages/2020_2021.js:2]: Hey this was the page which was commited before..Actually this page is not used.I didnt know how to remove that.Sorry.

File: src/pages/2020_2021.js (hunk 7 above)
Comment (ramumaha) [reply src/pages/2020_2021.js:34]: hey this page is not used.My mistake for commiting it.

File: src/pages/2020_2021.js (hunk 8 above)
Comment (ramumaha) [reply src/pages/2020_2021.js:88]: This page shouldn't have been included in the commit.Mistake from my side.

File: src/pages/events.js (hunk 9 above)
Comment (ramumaha) [reply src/pages/events.js:3]: Removed.

File: src/pages/past.js (hunk 10 above)
Comment (ramumaha) [reply src/pages/past.js:33]: This page should not been commited.My mistake.

**Pull request closed without merging**